### PR TITLE
Allow Model configuration 

### PR DESCRIPTION
**Pull request by matt-degraffenreid**
Addressing issue #55

### CODE REVIEW

File: src/utils/openAiModels.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const models = [
+  'gpt-3.5-turbo',
+  'text-davinci-003',
```
Comment (di-sukharev) [inline src/utils/openAiModels.ts:3]: davinci wont work as it's not the same openAI API endpoint, we use `CreateChatCompletion` for 3.5 and 4, but `CreateCompletion` for < 3.5

Comment (di-sukharev) [reply src/utils/openAiModels.ts:3]: let's remove it, 3.5 and 4 is enough

Comment (matt-degraffenreid) [reply src/utils/openAiModels.ts]: Went ahead and removed the other model leaving gpt-3.5-turbo and gpt-4 as the only valid options

**Comment by di-sukharev** [comment]
conflicts

**Pull request closed without merging**